tests/Paket.Tests/InstallModel/ProcessingSpecs.fs

```diff
@@ -154,7 +154,7 @@ let ``should understand reference folder``() =
         model.GetRuntimeAssemblies RuntimeGraph.Empty (Rid.Of "unix") (SinglePlatform (DotNetFramework FrameworkVersion.V4_6_3))
         |> Seq.map (fun f -> f.Library.Path)
     refs |> shouldNotContain @"..\System.Security.Cryptography.Algorithms\runtimes\win\lib\net46\System.Security.Cryptography.Algorithms.dll"
-    refs |> shouldContain @"..\System.Security.Cryptography.Algorithms\lib\netstandard1.6\System.Security.Cryptography.Algorithms.dll"
+    refs |> shouldNotContain @"..\System.Security.Cryptography.Algorithms\lib\netstandard1.6\System.Security.Cryptography.Algorithms.dll"
     refs |> shouldNotContain @"..\System.Security.Cryptography.Algorithms\ref\netstandard1.6\System.Security.Cryptography.Algorithms.dll"
```

Comment:
Can we add a similar test that checks if the rid stuff works, but now with a lower netstandard?
Because now all those changed to shouldnotcontain here so the test no longer tests that...